apps/server/src/server.ts

```diff
@@ -61,8 +61,9 @@ io.on('connection', (socket) => {
         rooms[roomId] = [];
       }
       rooms[roomId].push([username, socket.id]);
-      socket.to(roomId.toString()).emit('update_players', rooms[roomId]);
+      console.log(rooms[roomId]);
```

Comment:
A QUESTION: have you implement this? cuz I don't get this log when i join a room

Reply:
yes, this will log in the terminal. I have add for guest player. previously, it won't log if it's guest login.